icons/svg/new-manager-bold_32.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg width="16" height="16" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
```
What you can do is change the `width` and the `height` of the SVG and name it new-manager-bold_16

Reply:
I tried this already but it is not working properly. The SVG icon gets overlapped with the viewBox dimensions. It forced me to use the `viewBox="0 0 32 32"`

![Screen Recording 2023-02-23 at 4 02 28 PM](https://user-images.githubusercontent.com/77632250/220883063-ae368844-664e-454a-8b41-9d133127ccd4.gif)